unicon_backend/constants.py

```diff
@@ -1,15 +1,22 @@
 import os
-from typing import Final
+from typing import Final, Literal
 
 from dotenv import load_dotenv
 
 load_dotenv()
 
 
-def _get_env_var(name: str, default: str | None = None, required: bool = True):
+def _get_env_var(
```

Comment:
instead of using literals we can make use of generics and casting functions:

```py
def _get_env_var[T](
    name: str,
    default: T,
    cast: Callable[[str], T] = str,  # type: ignore
    required: bool = True,
):
    raw_value = os.getenv(name, default)
    if (raw_value is None) and required:
        raise ValueError(f"{name} environment variable not defined")
    return cast(raw_value) if raw_value is not None else raw_value  # type: ignore
```

Reply:
since now this function supports type conversions, we can edit initialization of `AMQP_TASK_QUEUE_MSG_TTL_SECS` (L33:34)

Reply:
Sure, will incorporate this.